src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -411,10 +413,27 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PropertyGroup>
       <_AssemblyInfoFile>$(IntermediateOutputPath)%(_SatelliteAssemblyResourceInputs.Culture)\$(TargetName).resources.cs</_AssemblyInfoFile>
       <_OutputAssembly>$(IntermediateOutputPath)%(_SatelliteAssemblyResourceInputs.Culture)\$(TargetName).resources.dll</_OutputAssembly>
+      <_CoreGenerateSatelliteAssembliesTemplateFile>$(IntermediateOutputPath)$(TargetName)$(TargetExt)</_CoreGenerateSatelliteAssembliesTemplateFile>
     </PropertyGroup>
 
+    <GetAssemblyInfo FilePath="$([System.IO.Path]::GetFullPath('$(_CoreGenerateSatelliteAssembliesTemplateFile)'))" >
```

Comment:
Should I get the full path of the main dll here and pass the full dll path to the task?